package-lock.json

```diff
@@ -3874,39 +3874,6 @@
         }
       }
     },
-    "@storybook/api": {
```

Comment:
Are these changes to `package-lock.json` intentional?

Reply:
yes

Reply:
Then I'm missing something. What does it have to do with this PR?

Reply:
it just needs to be done by  some  PR so I put it in this one to avoid triggering another build just to remove some unneeded things from the package-lock